praw/models/reddit/live.py

```diff
@@ -281,6 +281,45 @@ def close(self):
         url = API_PATH['live_close'].format(id=self.thread.id)
         self.thread._reddit.post(url)
 
+    def set(self, title, description='', nsfw=False, resources='',
+            **other_settings):
+        """Set settings for the live thread.
+
+        The following settings are supported:
+
+        :param title: The title of the live thread.
+        :param description: The live thread's description.
+        :param nsfw: Indicate whether this thread is not safe for
+            work.
+        :param resources: Markdown formatted information that is
+            useful for the live thread.
+        :returns: New :class:`.LiveThread` instance (updated settings
```

Comment:
Sorry I didn't bring this up earlier, but I want to make sure this behavior is consistent. The are two separate things:

1) At the moment there is only`Subreddit.mod.update` which always fetches the existing values before updating. It seems like we can use the same approach here -- I don't think the extra request to fetch existing values is that big a deal, and actually probably why I decided to remove the previously existing `set` function from the Subreddit (it was confusing having both and people often want the update behavior).

2) At the moment there is no return value from `Subreddit.mod.update`, nor is the existing object updated. I'm leaning toward wanting to update the existing object rather than return a new copy. If you're up for updating the thread's __dict__ with the new returned values that would be awesome. Otherwise, please remove the return and I'll take care of updating the Thread object along with doing that for Subreddit updates.

Please let me know should you require any clarification.